packages/flutter_analysis_options/README.md

```diff
@@ -0,0 +1,20 @@
+# Flutter Analysis Options
+
+This package contains the analysis lints used by the Flutter repositories:
+
+- [The Flutter framework](https://github.com/flutter/flutter)
+- [The Flutter engine](https://github.com/flutter/engine)
+- [Flutter plugins](https://github.com/flutter/plugins)
+- [Flutter packages](https://github.com/flutter/packages)
+
+Other projects are welcome to consume and use these rules as well. It contains
+two files:
+
+## analysis_options_base.yaml
+
+This is a subset of rules suitable for most Flutter projects.
+
+## analysis_options.yaml
+
+This contains all of the rules in `analysis_options_base.yaml`, plus additional
+rules that conform to the style guidelines for the Flutter repository.
```

Comment:
should it just import the former?

Reply:
analysis_options.yaml imports analysis_options_base.yaml

Reply:
(So yes, it should, and it does)

Reply:
Maybe "imports all of the rules" then?

Reply:
Done